src/AssetsPage/index.jsx

```diff
@@ -37,17 +36,14 @@ class AssetsPage extends React.Component {
     return (
       <div className={styles.assets}>
         <h2>Files & Uploads</h2>
-        <AssetsFilters />
-        <WrappedAssetsTable
-          assetsList={this.props.assetsList}
-        />
+        <WrappedAssetsFilters />
+        <WrappedAssetsTable />
       </div>
     );
   }
 }
 
 AssetsPage.propTypes = {
-  assetsList: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
The AssetsTable was subscribed to the assetsList slice of the Redux state and was receiving it as a prop from the AssetsPage, so I've removed assetsList from here in favor of the AssetsTable subscribing to the assetsList itself.